tests/conftest.py

```diff
@@ -201,26 +191,47 @@ def add_trade(trade):
             Trade.query.session.add(trade)
         else:
             LocalTrade.add_bt_trade(trade)
-
     # Simulate dry_run entries
     trade = mock_trade_1(fee)
     add_trade(trade)
-
     trade = mock_trade_2(fee)
     add_trade(trade)
-
     trade = mock_trade_3(fee)
     add_trade(trade)
-
     trade = mock_trade_4(fee)
     add_trade(trade)
-
```

Comment:
What about the one's that were formatted wrong before, I have format on save with autopep8 turned on, so it fixes parts [like this that were improperly formatted](https://i.imgur.com/0nYRHcc.png), I could turn format on save off and go back and change those if you want, but that seems like the wrong thing to do

Reply:
it's a difficult question to ask...
we're not using autopep8 - but pure flake8 for syntax validation at the moment - so to me - what isn't flagged by flake8 is not technically wrong.

Seems like autopep8 does some more things than flake8 - i'm not even sure if autopep8 aims to generate flake8 compliant code (it should - but don't know for sure) - as such it might even result in flake8 incompliant code (i hope it doesn't - but flake is checked after pytest) :laughing: